data_models/onelogin_data_model.yml

```diff
@@ -0,0 +1,19 @@
+AnalysisType: datamodel
+LogTypes: 
+  - OneLogin.Events
+DataModelID: onelogin.DataModel
+Filename: onelogin_data_model.py
+Enabled: true
+Mappings:
+  - Name: actor_user
+    Path: actor_user_name
+  - Name: admin_role
+    Path: privilege_name
+  - Name: event_type
+    Method: get_event_type
+  - Name: source_ip
+    Path: ipaddr
+  - Name: user
+    Path: user_name
+  - Name: user_agent
+    Path: user_agent
```

Comment:
I see, this is the unfortunate side affect of not supporting the default fields in the `.udm()` method. Still I think it was the right choice though.

Reply:
yea, it was one of the side effects. Another side effect (but on the positive side) is that `event_type` in Box.Event can differ for `.udm()` method vs `.get()` :)